### ISSUE TITLE
ImportError: cannot import name 'TypeGuard' from 'typing_extensions'

### ISSUE THREAD

**Issue by ZoranPandovski** [mindsdb/lightwood#643]
Starting MindsDB Container will throw bellow error because of the dependency of `black`.

```
  from lightwood.api.json_ai import generate_json_ai
  File "/opt/conda/lib/python3.8/site-packages/lightwood/api/json_ai.py", line 11, in <module>
    import black
  File "/opt/conda/lib/python3.8/site-packages/black/__init__.py", line 48, in <module>
    from black.files import find_project_root, find_pyproject_toml, parse_pyproject_toml
  File "/opt/conda/lib/python3.8/site-packages/black/files.py", line 26, in <module>
    from black.handle_ipynb_magics import jupyter_dependencies_are_installed
  File "/opt/conda/lib/python3.8/site-packages/black/handle_ipynb_magics.py", line 15, in <module>
    from typing_extensions import TypeGuard
ImportError: cannot import name 'TypeGuard' from 'typing_extensions' (/opt/conda/lib/python3.8/site-packages/typing_extensions.py)
```

**Comment by Davidnet** [comment]
How come that this appears ocassionally  in the container images?

**Comment by paxcema** [comment]
As an update: it's possible this was already fixed in #617. If not, then #650 will do the trick.
